src/validators/required.js

```diff
@@ -1,4 +1,4 @@
 import i18n from '@dhis2/d2-i18n'
 
 export const requiredMessage = i18n.t('Required')
-export const required = value => (!!value ? undefined : requiredMessage)
+export const required = value => (value ? undefined : requiredMessage)
```

Comment:
This kind of relates to the issue you brought up, but to be explicit we could call this requiredTruthy? That would make it very clear what it checks.

Reply:
I am already working on a validators branch in which `required` is going to be re-implemented. Resolving this because it has already been addressed, just not here...